deploy/rbac-clowdapp.yml

```diff
@@ -429,27 +408,74 @@ objects:
                 key: principal-proxy-source-cert
                 name: rbac-secret
                 optional: true
+          - name: PRINCIPAL_USER_DOMAIN
+            value: ${PRINCIPAL_USER_DOMAIN}
+          - name: PGSSLMODE
+            value: ${PGSSLMODE}
+          - name: CLOWDER_ENABLED
+            value: ${CLOWDER_ENABLED}
+          - name: CW_NULL_WORKAROUND
+            value: ${CW_NULL_WORKAROUND}
+          - name: GLITCHTIP_DSN
+            valueFrom:
+              secretKeyRef:
+                name: ${GLITCHTIP_SECRET}
+                key: dsn
+                optional: true
+          - name: MAX_SEED_THREADS
+            value: ${MAX_SEED_THREADS}
+          - name: ACCESS_CACHE_CONNECT_SIGNALS
+            value: 'False'
+          - name: NOTIFICATIONS_ENABLED
+            value: ${NOTIFICATIONS_ENABLED}
+          - name: NOTIFICATIONS_RH_ENABLED
+            value: ${NOTIFICATIONS_RH_ENABLED}
+          - name: KAFKA_ENABLED
+            value: ${KAFKA_ENABLED}
+          - name: NOTIFICATIONS_TOPIC
+            value: ${NOTIFICATIONS_TOPIC}
+          - name: EXTERNAL_SYNC_TOPIC
+            value: ${EXTERNAL_SYNC_TOPIC}
+          - name: EXTERNAL_CHROME_TOPIC
+            value: ${EXTERNAL_CHROME_TOPIC}
+          - name: MIGRATE_AND_SEED_ON_INIT
+            value: ${WORKER_MIGRATE_AND_SEED_ON_INIT}
```

Comment:
Should be SERVICE_MIGRATE_AND_SEED_ON_INIT?

Reply:
Good catch! Updated